packages/notifi-dapp-example/src/app/notifi/layout.tsx

```diff
@@ -11,15 +11,18 @@ export default function NotifiSingupLayout({
 }: {
   children: React.ReactNode;
 }) {
-  const { wallets, selectedWallet } = useWallets();
+  const { wallets, selectedWallet, isAuthenticationVerified } = useWallets();
   const pathname = usePathname();
   const router = useRouter();
 
   useEffect(() => {
-    if (!selectedWallet || !wallets[selectedWallet].walletKeys) {
+    if (
+      (!selectedWallet || !wallets[selectedWallet].walletKeys) &&
+      isAuthenticationVerified
```

Comment:
should we redirect to '/' when isAuthenticationVerified is false?

Reply:
no, we should only redirect users to home '/'  page if `isAuthenticationVerified` status is true